client/src/components/pages/MembersPage/MembersPage.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import styled from 'styled-components'
+import CreateForm from './components/CreateForm/CreateForm'
+import { ControlBar, ControlBarDisabled } from './components/ControlBar'
+import {
+  Visualisation,
+  VisualisationDisabled,
+} from './components/MembersVisualisation'
+
+import useForm from '../../../hooks/useForm'
+
+function MembersPage({
+  members,
+  setMembers,
+  bands,
+  setBands,
+  getMembers,
+  postMember,
+  patchMember,
+  deleteMember,
+}) {
+  const {
+    formIsVisible,
+    toggleForm,
+    newMembersInstruments,
+    addNewMembersInstrument,
+    removeNewMembersInstrument,
+    resetNewMembersInstruments,
+  } = useForm()
+
+  return (
+    <Grid>
+      {formIsVisible ? (
+        <>
+          <CreateForm
+            members={members}
+            setMembers={setMembers}
+            postMember={postMember}
+            newMembersInstruments={newMembersInstruments}
+            addNewMembersInstrument={addNewMembersInstrument}
+            removeNewMembersInstrument={removeNewMembersInstrument}
+            resetNewMembersInstruments={resetNewMembersInstruments}
+            bands={bands}
+            setBands={setBands}
+            toggleForm={toggleForm}
+          />
+          <VisualisationDisabled members={members} />
```

Comment:
Visualisation is a way to generic prefix, this could be any component in the view. Maybe rename it to something more descriptive.